src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -363,10 +363,14 @@ JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::ResizeDirection)
 // Possible SplitState values
 JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::SplitState)
 {
-    JSON_MAPPINGS(3) = {
+    JSON_MAPPINGS(7) = {
         pair_type{ "vertical", ValueType::Vertical },

```

Comment:
So you might be able to make some of this code easier! We could get rid of `SplitState::Horizontal` and `Vertical`, but _leave the serializations there_. So you'd have

```c++
        pair_type{ "vertical", ValueType::Right },
        pair_type{ "horizontal", ValueType::Down },
``` 

Then we wouldn't need to keep those legacy enum values internally, but we could keep them in the json deserialization.

Reply:
Oh, but we do use `SplitState::Horizontal/Vertical` for the actual internal "which way is this pane's divider". Hmm. Then maybe it makes more sense to have two types 
- one for "which direction has a user asked to split this pane in" ("up"/"down"/"left"/"right", "auto", "horizontal"=="down", "vertical"="right")
- the other is "which direction is this pane's divider" (`Horizontal`, `Vertical`)

Reply:
Would this end up with us serializing "right" as "horizontal" accidentally? That's why I didn't do that in the first place. If that isn't a problem I could split this into `SplitDirection` and `SplitState`.

Reply:
I did the refactor to make a separate `SplitDirection` that is used for SplitPaneArgs, and then an internal to pane `SplitState` that the direction gets converted to. I confirmed that `SplitDirection` deserializes "horizontal" and "vertical" as "down" and "right" directions appropriately.

Reply:
Also confirmed that a user setting for a `vertical` hotkey overrides (but displays as) a `right` split.

Reply:
I need to remember to not get enticed by "small" refactoring in the future since the changes are always bigger than I expect them to be.